source/manuals/programming-languages/nodejs/index.html.md.erb

```diff
@@ -4,41 +4,76 @@ title: Using Node.js at GDS
+## Introduction
 
-```
-❯ Running tasks for *.js
-   ✖ standard --fix
-     → standard: Use JavaScript Standard Style (https://standardjs.com)
-     git add
-✖ standard --fix found some errors. Please fix them and try committing again.
-  ~/nodejs-starter/index.js:5:9: 'hello' is assigned a value but never used.
+The guidance set out here should be followed in conjunction with the advice on
+the main [programming languages manual page](../../programming-languages.html).
 
-```
+The advice here is specifically about Node.js. Generic guidelines on writing
+code are out of scope.
+
+If you want to contribute to this document please see the [Updating this
+manual](#updating-this-manual) section below.
+
+Most guidelines listed here are recommendations, and the authors acknowledge
+that there will sometimes be valid exceptions.
+
+## Node versions
+
+> Only use Long Term Support (<abbr title="Long term support">LTS</abbr>)
+versions of Node.js.
+
+These are even-numbered versions (for example, Node.js 6.x or 8.x). However, it is important
+to keep an eye on the [Node.js LTS Schedule] as to when versions move in and out
+of LTS.
+
+> Don't use the `--harmony` or in-progress feature flags
+
+*Staged* or *in-progress* features aren't considered stable by the Node implementors.
 
-#### Editor plugins
-It’s useful to highlight lint errors before starting the commit process. To do this you can get live linting in your text editor of choice by installing one of [Standard’s plugins](https://standardjs.com/#are-there-text-editor-plugins). Users of [SublimeLinter](http://www.sublimelinter.com/) will find [StandardJS] will work out of the box using the default config.
+> Don't use language features beyond EcmaScript2016
 
-### Structure
-As well as having a consistent code style, it is important that our projects and repositories are structured in similar ways. This allows developers to find things quickly and easily. We want our code to be modular and reusable so our project structure should reflect this. Below are our four guiding principles: (thanks to [Rising Stack](https://blog.risingstack.com/) for this [blog post](https://blog.risingstack.com/node-hero-node-js-project-structure-tutorial/)).
+Even though ES2017 is final, some features aren't yet implemented in LTS versions. Check using [node.green](http://node.green/).
+
+## Source formatting and linting
+
+> Use the JavaScript Standard Style ([StandardJS])
+
+StandardJS includes a built in formatter, linter and automatic code fixer
+(auto-fix). To make sure everything stays clean we recommend running this tool
+before each commit. It is also possible to setup a pre-commit hook to automate this
+process. When StandardJS automatically fixes errors it has clear and concise
+messaging and it also tells you when an automatic fix isn't possible.
+
+We’ve encapsulated our recommended process into a separate
+[repository](https://github.com/alphagov/nodejs-starter) where everything is
```

Comment:
Would it be worth to point to alphagov/gds-nodejs-boilerplate and collaborate with Jon on the "everything is controlled"?

Reply:
Yes, I made some fixes on it yesterday and will point to it in this doc